
<launch>
	<arg name="robot_name" />
	<node name="TangentBug" pkg="cocktailparty_algorithm" type="tangentBug.py" ns="$(arg robot_name)" output ="screen" respawn="true">
		<param name= "robot_name" value="$(arg robot_name)" type="str"/>
	</node>
</launch>
